media/tracker_modules/xm.ksy

```diff
@@ -0,0 +1,280 @@
+meta:
+  id: xm
+  title: Extended Module
+  application:
+    - FastTracker 2
+    - Protracker
```

Comment:
Please list only applications that originated the format, not *all* applications that support it. It's like you list only "Adobe Photoshop" for .psd, not all possible viewers / raster image editors with kind of support .psd.